.pylintrc

```diff
@@ -69,7 +69,8 @@ disable=missing-docstring,anomalous-backslash-in-string,
     relative-import,redefined-outer-name,redefined-builtin,eval-used,
     exec-used,wildcard-import,
     inconsistent-return-statements,fixme,no-name-in-module,abstract-method,
-    useless-object-inheritance
+    useless-object-inheritance,no-else-break,unnecessary-comprehension,
+    invalid-overridden-method,invalid-overridden-method
```

Comment:
typo: invalid-overriden-method is disabled twice

Reply:
I'll remove the second occurence